test/CI-blackbox/java/renderer.js

```diff
@@ -0,0 +1,22 @@
+
+const {JavaGenerator} = require('../../../lib');
+const fs = require('fs');
+const path = require('path');
+/**
+ * Render all schema models
+ * @returns 
+ */
+async function renderModels() {
+  const input = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test.json')));
+  const generator = new JavaGenerator();
+  const generatedModels = await generator.generate(input);
+  const outputDir = path.resolve(__dirname, './output');
```

Comment:
how this will work when you run tests on local if the output will be already there? wouldn't there be a risk for some false/positives? In generator I did this https://github.com/asyncapi/generator/blob/master/test/integration.test.js#L18

Reply:
I guess that in general this part about creating output, you could move to a helper as I can see you have the same in every test

Reply:
Yea you are right on both points, let me see if I can't make them use the same function.

Reply:
So I moved everything to a single renderer. Regarding the naming, there is not since everything is rendered to a single file (`output.(js | java | ts)` and overwrite that file completely `false/positives` should not be possible.